CryptoPrices/Sources/MyCoin/Sources/MyCoin/TimeFrameSectionView/TimeFrameSection.swift

```diff
@@ -25,7 +25,6 @@ struct TimeFrameSection: View {
             HStack {
                 ForEach(timeFrameList) { item in
                     Button(item.title) {
```

Comment:
Feel free to ignore this 🙏 We can omit the conformance to `Identifiable` for the `TimeFrameItem`, i.e remove `let id = UUID()` by:
```swift
ForEach(timeFrameList, id: \.timeFrame) { item in
  // ...
}
```

Reply:
Thanks, I fixed it at https://github.com/nimblehq/nimble-crypto-ios/pull/99/commits/2fab7d69ff3151a83b4dd512405f946144981c0d